### DISCUSSION TITLE
If i have an existing semi built project, can I use gpt-pilot to complete it and make it release-ready?

### DISCUSSION THREAD

**Discussion by sr2echa** [Pythagora-io/gpt-pilot#558] [Q&A]
If i have an existing semi built project, can I use gpt-pilot to complete it and make it release-ready?

**Comment by benjaminearlevans** [comment]
Not at present

**Comment by dookiejones** [comment]
Sort of... You have to add the files on your own after creating the project in pilot. You can prompt it to ask you for files before creating them, though results on this have varied. Worst I have had to do is rename files. Its a bit hacky but once you can trick it into reading the contents of the files it will go from there.
